hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerHA.java

```diff
@@ -1270,6 +1279,7 @@ public void testListParts() throws Exception {
 
     // Stop leader OM, and then validate list parts.
     stopLeaderOM();
+    Thread.sleep(NODE_FAILURE_TIMEOUT * 2);
```

Comment:
What is this sleep for?

Reply:
Without this comment, `testListParts()` would show failure. Likely because I've set the ipc retry interval to a very low value.